services/core-web/src/actionCreators/noticeOfWorkActionCreator.js

```diff
@@ -0,0 +1,48 @@
+import { notification } from "antd";
+import { showLoading, hideLoading } from "react-redux-loading-bar";
+import { error, request, success } from "@common/actions/genericActions";
+import { ENVIRONMENT } from "@common/constants/environment";
+import { createRequestHeader } from "@common/utils/RequestHeaders";
+import CustomAxios from "@common/customAxios";
+import * as API from "../constants/API";
+import * as reducerTypes from "../constants/reducerTypes";
+
+export const getNoticeOfWorkApplicationDocument = (documentTypeCode) => {
+  CustomAxios()
+    .get(
+      `${ENVIRONMENT.apiUrl + API.NOTICE_OF_WORK_APPLICATION_DOCUMENT_TOKEN(documentTypeCode)}`,
+      createRequestHeader()
+    )
+    .then((response) => {
+      const token = { token: response.data.token_guid };
+      window.open(
+        `${ENVIRONMENT.apiUrl + API.NOTICE_OF_WORK_APPLICATION_DOCUMENT(documentTypeCode, token)}`,
+        "_blank"
+      );
+    });
+};
```

Comment:
This isn't actually an "action creator" but I placed it in this file to follow used patterns.